File: converted/classes_only/classes_only/WorldMarkers$Point.java
package zombie.iso;

class WorldMarkers$Point {
   float x;
   float y;

   WorldMarkers$Point(float var1, float var2) {
      this.x = var1;
      this.y = var2;
   }

   public WorldMarkers$Point set(float var1, float var2) {
      this.x = var1;
      this.y = var2;
      return this;
   }

   public boolean notInfinite() {
      return !Float.isInfinite(this.x) && !Float.isInfinite(this.y);
   }

   @Override
   public String toString() {
      return String.format("{%f, %f}", this.x, this.y);
   }
}

File: DoNotUse/include/zombie/iso/WorldMarkers$Point.cpp
#pragma once
#include <string>
#include <vector>
#include <memory>
#include <unordered_map>
#include <unordered_set>
#include <cstdint>

namespace zombie {
namespace iso {

class WorldMarkers {
public:
    float x;
    float y;

   WorldMarkers$Point(float var1, float var2) {
      this.x = var1;
      this.y = var2;
   }

   public WorldMarkers$Point set(float var1, float var2) {
      this.x = var1;
      this.y = var2;
    return this;
   }

    bool notInfinite() {
      return !float.isInfinite(this.x) && !float.isInfinite(this.y);
   }

    std::string toString() {
      return std::string.format("{%f, %f}", this.x, this.y);
   }
}
} // namespace iso
} // namespace zombie
